usrsctplib/netinet/sctp_pcb.c

```diff
@@ -5734,11 +5737,9 @@ sctp_free_assoc(struct sctp_inpcb *inp, struct sctp_tcb *stcb, int from_inpcbfre
 	 */
 	(void)SCTP_OS_TIMER_STOP(&asoc->strreset_timer.timer);
 	(void)SCTP_OS_TIMER_STOP(&asoc->dack_timer.timer);
-	(void)SCTP_OS_TIMER_STOP(&asoc->strreset_timer.timer);
```

Comment:
As far I understand here `SCTP_OS_TIMER_STOP` is used and not `sctp_timer_stop` because otherwise reference counter would be decremented twice, where first time it was decremented here:
https://github.com/sctplab/usrsctp/blob/0f1de1e6896523ac790ca0880fd6c91dc8a3d7cb/usrsctplib/netinet/sctp_pcb.c#L5555-L5568 is that correct?

Reply:
BTW, other than this block of `SCTP_OS_TIMER_STOP` usage there one more usage here:

https://github.com/sctplab/usrsctp/blob/0f1de1e6896523ac790ca0880fd6c91dc8a3d7cb/usrsctplib/netinet/sctp_pcb.c#L8072

Shouldn't `sctp_timer_stop` be used here or at least comment why not?

Reply:
That's correct. And yikes, can't believe I missed that, good catch. Fixed.